frontend/simple/sass/bulma_on_groupincome/notifications.scss

```diff
@@ -0,0 +1,6 @@
+.notification {
+	&.is-warning {
+		background-color: rgba($warning, 0.3);
```

Comment:
Yes, on my personal projects I use [stylelint](https://stylelint.io/), what do you think? I'll fix it it on this file

Reply:
👍 sounds great! a quick search also said that it could work with .vue files as well 👍👍 should we create a separate issue for that? how tricky is it to set it up?

Reply:
Not that tricky. Just search for the right configuration for our needs, install it, add it to our project and fix the current issues. Check #371 😄